docs/user/meta_multi_task_rl_exp.md

```diff
@@ -0,0 +1,304 @@
+# Run Meta-/Multi-Task RL Experiments
+
+This guide will walk you through how to run meta-/ multi-task RL experiments in garage.
+
+Similar to running all other experiments in garage, running meta-/ multi-task RL experiments generally involves steps such as:
+
+- defining the experiement with the `wrap_experiment` decorator;
+- constructing a `LocalRunner`;
+- constructing an environment;
+- constructing policy/ agorithm object(s).
+
+In meta-/ multi-task RL experiment, it revolves around solving multiple tasks and hence the construction of multiple / specific environment(s). Belows are a few environment wrappers commonly used in garage for meta-/ multi-task RL learning:
+
+- `MultiEnvWrapper`: a wrapper for mulitiple environments
+- `RL2Env`: a specific wrapper for RL2 environment
+
+Also, it's worth noting that [Meta-World](https://github.com/rlworkgroup/metaworld) provides a variety of benchmarked robotics tasks and can be extensively used in garage.
+
+## Meta-RL experiments
+
+The garage repository contains several meta-RL experiment examples. We will take a look at `te_ppo_metaworld_ml1_push.py` as below:
+
+```py
+from metaworld.benchmarks import ML1
```

Comment:
@krzentner @avnishn Do examples like this need to be changed to work with the new MetaWorld API?

Reply:
We haven't written the necessary `TaskSampler` yet, so they really can't.

Reply:
Let's just merge this for now, with the understanding that we still have a bunch of (re)writing to do once we get support for the new MetaWorld API in garage.